pymatgen/analysis/tests/test_quasirrho.py

```diff
@@ -0,0 +1,83 @@
+# coding: utf-8
+# Copyright (c) Pymatgen Development Team.
+# Distributed under the terms of the MIT License.
+"""
+Testing for quasirrho.py
+"""
+
+import os
+import unittest
+
+from pymatgen.analysis.quasirrho import QuasiRRHO
+from pymatgen.io.gaussian import GaussianOutput
+from pymatgen.io.qchem.outputs import QCOutput
+
+test_dir = os.path.join(os.path.dirname(__file__), "..", "..", "..",
```

Comment:
Instead of using module level test_dir, pls inherit from PymatgenTest for unit tests and then use self.test_dir instead. This will ensure correct behavior. Thanks.